addon/pods/date-range-picker/template.hbs

```diff
@@ -1,46 +1,51 @@
 <div class="dp-date-range-picker">
```

Comment:
I thought this too, but i didn't want to muddy the waters of this pr. In all fairness this line of code was never touched.